kernel_hardening_checker/__init__.py

```diff
@@ -219,8 +219,8 @@ def parse_sysctl_file(mode, parsed_options, fname):
         sys.exit(f'[!] ERROR: {fname} doesn\'t look like a sysctl output file, please try `sudo sysctl -a > {fname}`')
 
     # let's check the presence of a sysctl option available for root
-    if 'net.core.bpf_jit_harden' not in parsed_options and mode != 'json':
-        print(f'[!] WARNING: sysctl option "net.core.bpf_jit_harden" available for root is not found in {fname}, please try `sudo sysctl -a > {fname}`')
+    if 'kernel.cad_pid' not in parsed_options and mode != 'json':
```

Comment:
@cotequeiroz, here we need to choose a sysctl that exists in the kernel for quite a long time and is available only for root.
Could you please check these aspects for the `kernel.cad_pid` sysctl?
When did it first appear in the mainline kernel?

Reply:
This one should be good.  It is present since Linus initial git import, and does not appear to be guarded by any symbols besides sysctl being enabled.
https://github.com/torvalds/linux/blob/1da177e4c3f41524e886b7f1b8a0c1fc7321cac2/kernel/sysctl.c#L493

Reply:
Fixed this in https://github.com/a13xp0p0v/kernel-hardening-checker/pull/125/commits/0e0aa5f1516aeb4ffd9161d1d2c3479fe8caa39d:
changed `CONFIG_DEFAULT_INIT` to `CONFIG_LOCALVERSION`, which is older.